news/531.feature

```diff
@@ -0,0 +1 @@
+Add the ``api.content.iter_ancestors`` function for iterating over an object acquisition chain. @rohnsha0, @ale-rt
```

Comment:
```suggestion
Add the `api.content.iter_ancestors` function for iterating over an object acquisition chain. @rohnsha0, @ale-rt
```

Reply:
@davisagli the destination file is `CHANGES.rst`, so inline literals require double backticks. If we ever change it to `CHANGES.md`, then we can use Markdown syntax.

Reply:
@stevepiercy Oh! Let's go ahead and convert it to Markdown so it matches everything else.